lib/between_meals/repo.rb

```diff
@@ -50,89 +50,89 @@ def self.get(type, repo_path, logger)
     end
 
     def exists?
-      fail 'Not implemented'
+      fail "#{__method__} not implemented"
```

Comment:
Sigh. We're catching it somewhere higher and hiding the stacktrace, which needs to change, but not now